src/server/server.config.js

```diff
@@ -511,6 +517,20 @@ const mainNetworkId = get(ContractsAddress, `${network}-mainnet.networkId`, netw
 conf.set('ethereumMainnet', networks[mainNetworkId])
 conf.set('ethereum', networks[networkId])
 
+// phase options
+const phase = conf.get('phase')
```

Comment:
aren't we had PHASE_ONE / PHASE_ZERO env vars before ? or just on dapp ?
if there were no PHASE_ONE / PHASE_ZERO on server before - this isn't needed, just add  RELEASE_PHASE with default value = 0

Reply:
just on gooddap

Reply:
okay so remove please all stuff added, just keep RELEASE_PHASE (default=0)